applications/album-service/src/main/java/org/superbiz/moviefun/albums/AlbumServiceApplication.java

```diff
@@ -0,0 +1,43 @@
+package org.superbiz.moviefun.albums;
+
+import com.amazonaws.auth.AWSCredentials;
+import com.amazonaws.auth.BasicAWSCredentials;
+import com.amazonaws.services.s3.AmazonS3Client;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.superbiz.moviefun.blobstore.BlobStore;
+import org.superbiz.moviefun.blobstore.S3Store;
+import org.superbiz.moviefun.ServiceCredentials;
+
+@SpringBootApplication
+public class AlbumServiceApplication {
+    public static void main(String... args) {
+        SpringApplication.run(AlbumServiceApplication.class, args);
+    }
+
+    @Bean
+    ServiceCredentials serviceCredentials(@Value("${vcap.services}") String vcapServices) {
+        return new ServiceCredentials(vcapServices);
+    }
+
+    @Bean
+    public BlobStore blobStore(
+            ServiceCredentials serviceCredentials,
+            @Value("${vcap.services.photo-storage.credentials.endpoint:#{null}}") String endpoint
+    ) {
+        String photoStorageAccessKeyId = serviceCredentials.getCredential("photo-storage", "user-provided", "access_key_id");
+        String photoStorageSecretKey = serviceCredentials.getCredential("photo-storage", "user-provided", "secret_access_key");
+        String photoStorageBucket = serviceCredentials.getCredential("photo-storage", "user-provided", "bucket");
+
+        AWSCredentials credentials = new BasicAWSCredentials(photoStorageAccessKeyId, photoStorageSecretKey);
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

If possible, use temporary security credentials (IAM roles) instead of long-term access keys.
Long-term access keys, such as those associated with IAM users and AWS account root users, remain valid until you manually revoke them. However, temporary security credentials obtained through IAM roles and other features of the AWS Security Token Service expire after a short period of time. Use temporary security credentials to help reduce your risk in case credentials are accidentally exposed.
Learn more about [best practices for managing AWS access keys](https://docs.aws.amazon.com/general/latest/gr/aws-access-keys-best-practices.html).